config/config.yaml

```diff
@@ -529,9 +532,9 @@ clouds:
             maxCount: 6
             vmSize: 'Standard_D4s_v3'
             osDiskSizeGB: 100
-            azCount: 3
+            azCount: {{ .ev2.availabilityZoneCount }}
           infraAgentPool:
-            azCount: 1
+            azCount: {{ .ev2.availabilityZoneCount }}
```

Comment:
i think for this we should remain at hardcoded `1` in DEV to save some CPU quota in busy regions

Reply:
or at least for the `pers` env